config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   root 'static_pages#index'
 
+  get '*path', to: "static_pages#index"
+
```

Comment:
i strongly encourage you to not use `*path`, as it allows all get requests regardless of path to be routed to this controller. Better to explicitly name the paths you want to allow in your app.